setup.cfg

```
@@ -27,7 +27,7 @@ package_dir =
     = src
 packages = find_namespace:
 install_requires =
-    stactools == 0.2.3
+    stactools
```

```suggestion
    stactools >= 0.2.3
```
Using an older version than the package was developed against could be risky though, yes?